oauth2_provider/models.py

```diff
@@ -216,6 +238,8 @@ def allow_scopes(self, scopes):
         """
         if not scopes:
             return True
+        if isinstance(scopes, six.string_types):
```

Comment:
if it is not a string it has to be a list, so then there is no need to split it into a list.